frontend/src/components/Themes/ThemeCard.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+
+const bgColors = {
+  "Explorar": "#00BFFF",
+  "Explorando": "#FFD700",
+  "Explorado": "#32CD32"
+};
+
+export default class ThemeCard extends Component {
```

Comment:
Fix the identation. Remember that the pattern that we are using is tabs of 2 spaces for React code.